Cargo.toml

```diff
@@ -47,6 +47,16 @@ zksync-error-codegen = { git = "https://github.com/matter-labs/zksync-error", re
 zksync-error-description = { git = "https://github.com/matter-labs/zksync-error", rev = "54e0671189a5bc2ce88e72c36334f98fbbdd228d" }
 zk_evm_abstractions = "=0.152.3"
 
+#########################
+# ZKsyncOs dependencies  #
+#########################
+forward_system = { git = "https://github.com/matter-labs/zksync-os.git", tag = "0.0.2"}
+basic_system = { git = "https://github.com/matter-labs/zksync-os.git", tag = "0.0.2"}
+zk_ee = { git = "https://github.com/matter-labs/zksync-os.git", tag = "0.0.2"}
+system_hooks = { git = "https://github.com/matter-labs/zksync-os.git", tag = "0.0.2"}
+zksync_os_api = { git = "https://github.com/matter-labs/zksync-os.git", tag = "0.0.2" }
+rig = { git = "https://github.com/matter-labs/zksync-os.git", tag = "0.0.2" }
```

Comment:
JFYI zksync-os is at 0.0.4 already so we should upgrade when we can. Can be done in a follow-up PR though ofc